Sources/Extensions/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -197,6 +197,17 @@ public extension Sequence where Element: Hashable {
         return false
     }
 
+    /// SwifterSwift: Getting the duplicated elements in a sequence.
+    ///
+    ///     [1, 1, 2, 2, 3, 3, 3, 4, 5].duplicates() -> [1, 2, 3])
+    ///     ["h", "e", "l", "l", "o"].duplicates() -> ["l"])
+    ///
+    /// - Returns: An array of duplicated elements.
+    ///
+    public func duplicates() -> [Element] {
+        let countedSet = NSCountedSet(array: Array(self))
```

Comment:
I'm still not sure about relying on a Foundation API here. To me, there are some downsides on doing that way. Also if the duplicates are returned single, should we return a Set instead of an Array?

Reply:
I think an `Array` is a nicer API and is likely what the Swift Standard Library would choose to return here. 

I do agree we should probably use an implementation similar to that of the `removeDuplicates`extension on `Array`. Lower complexity, no Foundation, less type conversion

Reply:
Ok! Let me fix that...